pytorch_lightning/core/step_result.py

```diff
@@ -395,6 +395,11 @@ def detach(self):
             if isinstance(v, torch.Tensor):
                 self.__setitem__(k, v.detach())
 
+    def cpu(self):
+        for k, v in self.items():
```

Comment:
can we add a simple docstring what it does?
```suggestion
        """Move all self attributes to CPU."""
        for k, v in self.items():
```